### DISCUSSION TITLE
Centered elements move to the right

### DISCUSSION THREAD

**Discussion by Dhanushbd03** [valinet/ExplorerPatcher#3275] [General]
![image](https://github.com/valinet/ExplorerPatcher/assets/119742887/db276fbb-fbc7-413d-98e8-e34942ce2269)

check my primary taskbar alignment is centered with start but a unnecessary space is created between start and other elements
Please checkout this issue

**Comment by pyrates999** [comment]
please see this: https://github.com/valinet/ExplorerPatcher/wiki/ExplorerPatcher's-taskbar-implementation
